feature/eventmap/src/commonTest/kotlin/io/github/droidkaigi/confsched/eventmap/EventMapScreenTest.kt

```diff
@@ -0,0 +1,147 @@
+                    checks = {
+                        checkEventMapItemKoala()
+                    },
+                )
+            }
+            doIt {
+                scrollToLadybugRoomEvent()
+            }
+            itShould("ensure that the room types for Ladybug are displayed.") {
+                captureScreenWithChecks(
+                    checks = {
+                        checkEventMapItemLadybug()
+                    },
+                )
+            }
+            doIt {
+                scrollToMeerkatRoomEvent()
+            }
+            itShould("ensure that the room types for Meerkat are displayed.") {
+                captureScreenWithChecks(
+                    checks = {
+                        checkEventMapItemMeerkat()
+                    },
+                )
+            }
+            doIt {
+                scrollToNarwhalRoomEvent()
+            }
+            itShould("ensure that the room types for Narwhal are displayed.") {
+                captureScreenWithChecks(
+                    checks = {
+                        checkEventMapItemNarwhal()
+                    },
+                )
+            }
+        }
+        // FIXME In the test code, DefaultErrorFallbackContent is not displayed even when an error occurs, so this test cannot pass as is.
+        // FIXME In another test, DefaultErrorFallbackContent is displayed when an error occurs, and the test passes, so I believe the cause lies somewhere in the EventMap test code.
+//        describe("when server is error") {
+//            doIt {
+//                setupEventMapServer(ServerStatus.Error)
+//            }
+//            describe("when launch") {
+//                doIt {
+//                    setupEventMapScreenContent()
+//                }
+//                describe("after loading") {
+//                    doIt {
+//                        waitFor5Seconds()
+//                    }
+//                    itShould("show error message") {
+//                        captureScreenWithChecks {
+//                            checkErrorFallbackDisplayed()
+//                        }
+//                    }
+//                    describe("click retry after server gets operational") {
+//                        doIt {
+//                            setupEventMapServer(ServerStatus.Operational)
+//                            clickRetryButton()
+//                        }
+//                        describe("after waiting for 5 seconds") {
+//                            doIt {
+//                                waitFor5Seconds()
+//                            }
+//                            itShould("show event map") {
+//                                captureScreenWithChecks {
+//                                    checkEventMapDescriptionDisplayed()
+//                                }
+//                            }
+//                        }
+//                    }
+//                }
+//            }
+//        }
```

Comment:
- We implemented this by referencing the implementation of StaffScreenTest, which was already in progress.
    - https://github.com/DroidKaigi/conference-app-2025/pull/217/files#diff-6847480399605f1b89d01b694e311f95402788d80f24bef2b98728a6d5b8ec1f
- However, in EventMapScreen, even when an IOException is thrown, the DefaultErrorFallbackScreen is not displayed, and this test currently fails.
- This issue requires a separate resolution.

Reply:
- As shown below, in the existing FakeEventMapApiClient, an IO Exception is thrown when ErrorStatus occurs.

https://github.com/DroidKaigi/conference-app-2025/blob/bfb8bfafeb34a1a1bc21d30724ae43f970c4ac60/core/data/src/commonMain/kotlin/io/github/droidkaigi/confsched/data/eventmap/FakeEventMapApiClient.kt#L17-L22

Reply:
There were two causes.
1. The existing FakeEventMapApiClient was not injected as a singleton in DataScope.
2. Unnecessary Provides were included in TestAppGraph.
https://github.com/DroidKaigi/conference-app-2025/pull/222/commits/3f38890dfb1f9600c8f8abd22b95a6e83a865077